sys/riscv/conf/std.NODEBUG

```diff
@@ -0,0 +1,12 @@
+# NODEBUG -- Common kernel config options for WITNESS and INVARIANTS free kernels
+
+nooptions       INVARIANTS
+nooptions       INVARIANT_SUPPORT
+nooptions       WITNESS
+nooptions       WITNESS_SKIPSPIN
+nooptions       BUF_TRACKING
+nooptions       DEADLKRES
+nooptions       FULL_BUF_TRACKING
```

Comment:
seems like there is an inconsistent mixture of tabs and spaces here?

Reply:
Looks like it was copied from the original file FWIW.

Reply:
Yes, and that's shared across all architectures, as the early GENERIC-NODEBUGs used only spaces and somewhere along the way people started using tabs instead as they should have done in the first place.